zuul.d/jobs.yaml

```diff
@@ -1236,3 +1236,36 @@
       - moo_github_account
     vars:
       is_gpu_enabled: true
+
+- job:
+    name: tensorflow-benchmarks-volcano/kubeflow-k8s-v1.14.0
+    parent: init-test
+    description: |
+      Test benchmarks of tensorflow + volcano/kubeflow on k8s cluster
+    run: playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/run.yaml
+    post-run: playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/post.yaml
+    timeout: 72000
```

Comment:
The timeout is so long, and we should set 'max-job-timeout' in zuul/main.yaml, now it's 36000?

Reply:
need to enlarge the setting in zuu/main.yaml, I still afraid 72000 is not enough long for this job :)